apps/allocations/app/components/Card/Budget.js

```diff
@@ -90,40 +91,38 @@ const Budget = ({
             <IconProhibited />
             <ActionLabel>Deactivate</ActionLabel>
           </ContextMenuItem>
-        </ContextMenu>
-      </MenuContainer>
-      <CardTitle color={`${theme.content}`}>{name}</CardTitle>
-      <StatsContainer>
-        <StyledStats>
-          <StatsValueBig color={`${theme.contentSecondary}`}>
-            <Text>{`${tokenAmount} ${token.symbol} per period`}</Text>
-          </StatsValueBig>
-          <StatsValueBig css={{ paddingTop: '24px' }}>
-            <ProgressBar
-              color={`${theme.accentEnd}`}
-              value={allocated}
-            />
-          </StatsValueBig>
-          <StatsValueSmall
-            css={{
-              color: theme.content,
-              paddingTop: '8px',
-            }}
-          >
-            
-            <Text>{`${tokensRemaining} ${token.symbol} below limit`}</Text>
-          </StatsValueSmall>
-          <StatsValueSmall
-            css={{
-              color: theme.contentSecondary,
-              paddingTop: '4px',
-            }}
-          >
-            <Text>{`${tokensRemaining.div(tokenAmount).times(100)}% remaining`}</Text>
-          </StatsValueSmall>
-        </StyledStats>
-      </StatsContainer>
-    </StyledCard>
+        </React.Fragment>
```

Comment:
Looks like `ContextMenuItem` is the only child of `React.Fragment` here, which makes `React.Fragment` unnecessary.